src/appsettings.cpp

```diff
@@ -41,3 +42,17 @@ void AppSettings::setSendByEnter(bool sendByEnter)
         emit sendByEnterChanged();
     }
 }
+
+bool AppSettings::showStickersAsImages() const
+{
+    return settings.value(KEY_SHOW_STICKERS_AS_IMAGES, false).toBool();
```

Comment:
Default should be true.

Reply:
Hmm... but you said you don't want to surprise people!

Reply:
Oh well, it's your code. If you want to surprise, let it be a surprise.